hoomd/md/pair/aniso.py

```diff
@@ -316,6 +316,20 @@ class ALJ(AnisotropicPair):
     * 3:
       All interactions include attractive and repulsive components.
 
+    .. list-table:: Set alpha based on range of the center-center and contact-contact interactions.
```

Comment:
What do you think of this format? I think it's easier to read. Should it replace the list form above this?
<img width="637" alt="image" src="https://user-images.githubusercontent.com/21282461/196770753-7f40c8fa-82a2-4525-ab72-d939c975b237.png">